opentelemetry-sdk/Cargo.toml

```diff
@@ -18,6 +18,7 @@ crossbeam-channel = { version = "0.5", optional = true }
 futures-channel = "0.3"
 futures-executor = "0.3"
 futures-util = { version = "0.3.17", default-features = false, features = ["std", "sink", "async-await-macro"] }
+lazy_static = "1.4"
```

Comment:
nit: Can use https://docs.rs/once_cell/latest/once_cell/#lazy-initialized-global-data instead of an additional dependency

Reply:
will check it out.  thanks!

Reply:
Yes -- in general, lazy_static should no longer be used. once_cell is on a path into the standard library.

Reply:
Good to know!  PR is updated with `once_cell` now.